src/main/java/me/moodcat/database/entities/User.java

```diff
@@ -63,4 +71,14 @@
     @Column(name = "access_token", nullable = true)
     private String accessToken;
 
+    /**
+     * Updates the points for the user.
+     * 
+     * @param addition
+     *            The amount of points the user gained
+     */
+    public void increment(Integer addition) {
+        this.points = this.getPoints() + addition;
```

Comment:
Should be generated by lombok. if we use a getter here, we might want to use a setter as well.
```java
this.setPoints(this.getPoints() + addition);
```